solr/conf/schema.xml

```diff
@@ -276,10 +273,10 @@
     <fieldType name="location_rpt" class="solr.SpatialRecursivePrefixTreeFieldType"
                geo="true" distErrPct="0.025" maxDistErr="0.001" distanceUnits="kilometers"/>
 
-    <field name="bbox" type="bbox" />
-    <fieldType name="bbox" class="solr.BBoxField"
-               geo="true" distanceUnits="kilometers" numberType="_bbox_coord" storeSubFields="false"/>
-    <fieldType name="_bbox_coord" class="solr.DoublePointField" docValues="true" stored="false"/>
+    <!--<field name="bbox" type="bbox" />-->
+    <!--<fieldType name="bbox" class="solr.BBoxField"-->
+    <!--           geo="true" distanceUnits="kilometers" numberType="_bbox_coord" storeSubFields="false"/>-->
+    <!--<fieldType name="_bbox_coord" class="solr.DoublePointField" docValues="true" stored="false"/>-->
 
```

Comment:
We seem to have commented out the bbox field (here and in dynamic fields) and the fields being copied to suggest. Just thought I'd run them by you before approving the PR.

Reply:
IIRC they broke the more recent solr but I'll double check before you approve.